README.md

```diff
@@ -109,15 +122,15 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
-
+calling a function invokes that function immediately. referencinga function is for when you need it to run at a certain point.
 ```
```

Comment:
:+1: More specifically, referencing a function returns its definition. Its that definition that we, for example, would pass as a callback argument to an event listener.